### DISCUSSION TITLE
Update db8lastModified

### DISCUSSION THREAD

**Discussion by Alain0842** [community/community#72043] [Packages]
### Select Topic Area

Bug

### Body

The file [mod_db8sitelastmodified](https://github.com/pe7er/mod_db8sitelastmodified/tree/master) update.xml

don't work with joomla 4.4.0

<targetplatform name="joomla" version="4\.(0|1|2)"/>  ??

Alain

**Comment by esurov** [comment]
Dear @Alain0842 ,
As far as I can see you have already reported this issue to the author of `mod db8 site last modified` extension and he is going to fix it.
Well done! Thanks for your contribution!
